.golangci.yml

```diff
@@ -49,3 +77,6 @@ linters-settings:
 #     disabled-checks:
 #       - wrapperFunc
 #       - commentFormatting # https://github.com/go-critic/go-critic/issues/755
+
+service:
```

Comment:
this is done for now, will test tomorrow if this can be removed, the bot was not upgraded in accordance with the breaking change